ci-bootstrap.sh

```diff
@@ -101,3 +101,20 @@ else
   colored_text "gcc version"
   gcc --version
 fi
+
+colored_text "plist integer type check"
+ret=0
+while read -r line; do
+  if [ "$(grep -z '<real>(.\|\s)*</real>' "${line}")" != "" ]; then
+    echo "Please change <real>*</real> back to <integer>*</integer> in ${line}"
+    # FIXME: Find better ways to patch the files
+    perl -pi -e 's/<real>/<integer>/g' "${line}"
```

Comment:
can't `sed` be used?

Reply:
> can't `sed` be used?

e.g. `sed -E 's/[real]?[real]+/integer/g' <<< "<real>12345678</real>"` returns `<integer>12345678</integer>`

Reply:
After digging on regex101.com `sed -i '.bak' -E 's/(real)|(\/real)/integer/g' layout11.xml` can be used. Eventually a for cycle between all the `Resources` can be used and additionally replace the file content

Reply:
That is too lax, it will match against instances that are not XML tags. `sed -i '.bak' -E 's/<(\/)?real>/<\1integer>/g' "${line}"` is better.

Reply:
https://github.com/acidanthera/ocbuild/commit/1ac9f65e6154325b5f31123be860519848115a62#diff-29e7693adc138446b9a0e4c91dec95539d36e13b5e493d8ab6a16d4a4899e0c6